infra/examples-dev/aws-google-workspace/variables.tf

```diff
@@ -49,6 +49,12 @@ variable "force_bundle" {
   default     = false
 }
 
+variable "enable_testing" {
```

Comment:
Maybe I'd rename it as `support_deployment_testing` or something like that, `enable_testing` seems too generic